<launch>
	<rosparam file = "$(find modellazione)/config/mission.yaml"/>
	<rosparam file = "$(find modellazione)/config/constant.yaml"/>
	
	<node name ="model" pkg="modellazione" type="model" output="screen"/>
	<node name="ahrs" 	pkg="modellazione" type="ahrs_sensor" output="screen"/>
	<node name="depth" 	pkg="modellazione" type="depth_sensor" output="screen"/>
	<node name="gps" 	pkg="modellazione" type="gps_sensor" output="screen"/>
	<node name="usbl"	pkg="modellazione" type="usbl_sensor" output="screen"/>
	<node name="dvl" 	pkg="modellazione" type="dvl_sensor" output="screen"/>
	<node name="control" pkg="modellazione" type="fake_control" output="screen"/>
	<node name="nav" 	pkg="modellazione" type="fake_nav" output="screen"/>
	<node name="plot"	pkg="rqt_plot" type="rqt_plot"/>
</launch>